weave-js/src/components/PagePanelComponents/Home/Browse3.tsx

```diff
@@ -691,6 +707,8 @@ const CallsPageBinding = () => {
       onFilterUpdate={onFilterUpdate}
       columnVisibilityModel={columnVisibilityModel}
       setColumnVisibilityModel={setColumnVisibilityModel}
+      sortModel={sortModel}
+      setSortModel={setSortModel}
```

Comment:
we might end up wanting a context for this (?)

Reply:
Yes, I expect we'll eventually want either a container object or a context since the number of props is going to keep ballooning as we add more things to state (filters should come soon, but could also be column order, column pinning, row selection, etc.) I need to think about what would be nicest.